src/RazorConsole.Gallery/Components/ScrollableGallery.razor

```diff
@@ -0,0 +1,120 @@
+﻿@using Spectre.Console
+@using Markup = RazorConsole.Components.Markup
+@using Panel = RazorConsole.Components.Panel
+@using Rows = RazorConsole.Components.Rows
+@namespace RazorConsole.Gallery.Components
+
+<Rows>
+    <Markup Content="Scrollable component that adds ability to scroll through items and render limited count."
+            Foreground="@Color.Grey70"/>
+    <Markup Content="Source Code: Scrollable.razor"
+            Foreground="@Color.DeepSkyBlue1" Decoration="@Decoration.Underline"
+            link="@DocumentationUrl"/>
+    <Newline/>
+
+    <Columns>
```

Comment:
Pls use Rows instead of Columns so the code and preview are placed upper and lower.